noxfile.py

```diff
@@ -255,6 +262,20 @@ def slotscheck(session: nox.Session) -> None:
     session.run("python", "-m", "slotscheck", "--verbose", "-m", "disnake")
 
 
+@nox.session(requires=["check-manifest"])
+def build(session: nox.Session) -> None:
+    """Build a dist."""
+    install_deps(session)
+    import pathlib
+
+    dist_path = pathlib.Path("dist")
+    if dist_path.exists():
+        import shutil
```

Comment:
Should be a top-level import. Performance-wise it doesn't matter, nox already imports shutil so this is a nanosecond difference at best